scripts/setup.sh

```diff
@@ -175,6 +174,19 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Download and install yarn.
+echo Checking if yarn is installed in $TOOLS_DIR
+if [ ! -d "$YARN_PATH" ]; then
+  echo Installing yarn
+
+  YARN_VERSION=v1.17.3
+  YARN_FILE_NAME=yarn-$YARN_VERSION.tar.gz
+
+  wget https://github.com/yarnpkg/yarn/releases/download/$YARN_VERSION/$YARN_FILE_NAME
```

Comment:
I think we prefer curl

Reply:
That's fine, I just did this since that's what the official docs say. Do you know the reasoning why we have that preference?

Reply:
IIRC the issue was that wget didn't work out of the box on some systems, though I can't remember which systems those were (it was Mac or some flavour of Linux).

Reply:
Ah, okay.